charts/llm-d-infra/values.yaml

```diff
@@ -60,7 +60,7 @@ gateway:
   nameOverride: ""
 
   # -- Gateway class that determines the backend used
-  # Currently supported values: "kgateway" or "istio"
+  # Currently supported values: "kgateway", "istio", or "gke-l7-regional-external-managed"
```

Comment:
This is tricky... updating the values at all will generate a new schema meaning it technically should have a chart bump, which we dont want because we dont want to re-release. I guess the best path forward is to update values and schema but not chart bump, meaning if anyone tries to use these charts remotely it wont work / will be different from cloning them. In this example thats not that big of a deal because to use the quickstarts they have to clone the repo so we don't really care too much. Those schema changes (just description) will be captured next round

Reply:
No I think lets keep it and ill add the schema changes I just wanted to explain why this is not best practice but we can do it